packages/capa-explorer-web.vm/capa-explorer-web.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>capa-explorer-web.vm</id>
+    <version>1.0.0</version>
+    <authors>Mandiant FLARE</authors>
```

Comment:
Completely optional: @fariss I think you should be the author. The author in the nuspec is the tool author and you have implemented most of capa Explorer Web. :wink:

Reply:
Sounds good 👍, I've addressed these comments.